oat-sa/extension-tao-itemqti
# Fix/aut 1906/allow score maxscore

related to https://oat-sa.atlassian.net/browse/AUT-1906

### Description
When response processing is turned off for interaction, SCORE and MAXSCORE outcome declarations are ignored and not saved into item xml, this PR allows them to be entered manually and saved

### How to test

- run unit tests
- check with related PR (https://github.com/oat-sa/tao-item-runner-qti-fe/pull/308)
